AdvancedDLSupport/SymbolTransformation/SymbolTransformer.cs

```diff
@@ -105,29 +105,64 @@ private string Transform
                 }
                 case Pascalize:
                 {
-                    return concatenated.Pascalize();
+                    return PascalizeValue(concatenated);
                 }
                 case Camelize:
                 {
-                    return concatenated.Camelize();
+                    return CamelizeValue(concatenated);
                 }
                 case Underscore:
                 {
-                    return concatenated.Underscore();
+                    return UnderscoreValue(concatenated);
                 }
                 case Dasherize:
                 {
-                    return concatenated.Dasherize();
+                    return DasherizeValue(concatenated);
                 }
                 case Kebaberize:
                 {
-                    return concatenated.Kebaberize();
+                    return KebaberizeValue(concatenated);
                 }
                 default:
                 {
                     throw new ArgumentOutOfRangeException(nameof(method), method, null);
                 }
             }
         }
+
+        private string CamelizeValue(string input)
```

Comment:
Add missing XMLDocs to these functions. As per library convention, even private functions should be documented in full.